rom-hacking/scripting/animation-commands/ac-dir-d.md

```diff
@@ -0,0 +1,30 @@
+# (775) _AC_DIR_D
+
+## Effect
+
+Turns the actor of the animation down over a given amount of frames.
+
+If the second argument is provided, the actor will invert the turn.
+
+Starts the stepping sound effect.
+
+## Syntax
+
+```c
+_AC_DIR_D(frames, invert?)
+```
+
+| Argument | Description | Types | Required |
+| - | - | - | - |
+| **frames** | The amount of frames | Float | Required |
```

Comment:
With _TIME_WAIT, I described the time a bit differently. Might be good to uniformize it like that. In the "Effect" section as well.